Makefile

```diff
@@ -7,6 +10,16 @@ run: _data/versions.json
 		jekyll/jekyll -- \
 		jekyll serve --livereload --livereload-port 4001
 
+run_docs_local: local_docs_dir _data/versions.json
```

Comment:
Nitpick: I generally use hyphens for things that humans will type, because it takes fewer keystrokes than `_`. I wouldn't use `node_modules` as an example in this case, because it is only named that way because that's what the directory it builds is called.